src/player.cpp

```diff
@@ -95,8 +95,7 @@ void game::init_morale()
     _("Killed Friend"),
     _("Guilty about Killing"),
     _("Guilty about Mutilating Corpse"),
-    _("Chimerical Mutation"),
-    _("Fey Mutation"),
+    _("Mutation"),
```

Comment:
I can keep them if you are alright with the added complexity in a if statement.  I'm not sure why add_morale isn't designed to accept strings(performance reasons?)